### PR TITLE
added x-refs to image, jenkins-ignore

### PR DESCRIPTION
**Pull request by rosemarymarano**
Added x-refs to obtaining the FMW image in two places. 
Also, removed a repeated section in the FMW Infra domains doc.

### CODE REVIEW

File: docs-source/content/userguide/managing-domains/fmw-infra/_index.md [hunk 1]
```diff
@@ -88,38 +88,6 @@ Additional information about using this image is available in the
 [Oracle Container Registry](https://container-registry.oracle.com).
 
 
-#### Obtaining the FMW Infrastructure Docker image
```
Comment (rjeberhard) [inline docs-source/content/userguide/managing-domains/fmw-infra/_index.md]: Is this duplicated text?  I don't understand why you are deleting this content... :)

Comment (rosemarymarano) [reply docs-source/content/userguide/managing-domains/fmw-infra/_index.md]: The "Obtaining the FMW Infrastructure Docker image" section was duplicated in my branch. However, now when I look in Master, I'm not seeing it that way. Do you?

Comment (rosemarymarano) [reply docs-source/content/userguide/managing-domains/fmw-infra/_index.md]: That sounds like a good idea. Perhaps, you can create a JIRA for it and assign it to me. I'd be glad to do the work when I am able.

Comment (rjeberhard) [reply docs-source/content/userguide/managing-domains/fmw-infra/_index.md]: Okay, yes, I also see it duplicated.

**Pull request merged**